posydon/popsyn/synthetic_population.py

```diff
@@ -224,8 +224,84 @@ def merge_parallel_runs(self, pop):
 # Helper classes #
 ##################
 
+class DFInterface:
+    """A class to handle the interface between the population file and the History and Oneline classes."""
+    
+    def __init__():
+        pass
```

Comment:
The `__init__` should make sure that all elements, this class needs are created: `self.filename` and `self.chunksize`. Maybe a `self.verbose` will be useful, like the subclasses have.

Reply:
Added